docs/user/hbase/usage.rst

```diff
@@ -9,13 +12,35 @@ including ``hbase.zookeeper.quorum`` and ``hbase.zookeeper.property.clientPort``
 .. code-block:: java
 
     Map<String, Serializable> parameters = new HashMap<>();
-    parameters.put("bigtable.table.name", "geomesa");
+    parameters.put("hbase.catalog", "geomesa");
     org.geotools.data.DataStore dataStore =
         org.geotools.data.DataStoreFinder.getDataStore(parameters);
 
-The data store takes one parameter:
+.. _hbase_parameters:
+
+Parameters
+----------
+
+The data store takes several parameters (required parameters are marked with ``*``):
 
-* ``bigtable.table.name`` - the name of the HBase table that stores feature type data
+====================================== ======= ====================================================================================
+Parameter                              Type    Description
+====================================== ======= ====================================================================================
+``hbase.catalog *``                    String  The name of the GeoMesa catalog table (previously ``bigtable.table.name``)
+``hbase.coprocessor.url``              String  Path to the GeoMesa jar containing coprocessors, for auto registration
+``hbase.remote.filtering``             Boolean Can be used to disable remote filtering and coprocessors, for environments
+                                               where custom code can't be installed
+``hbase.security.enabled``             Boolean Enable HBase security (visibilities)
+``geomesa.security.auths``             String  Comma-delimited superset of authorizations that will be used for queries
+``geomesa.security.force-empty-auths`` Boolean Forces authorizations to be empty
+``geomesa.query.audit``                Boolean Audit queries being run. Queries will be written to a log file
+``geomesa.query.timeout``              Integer The max time (in seconds) a query will be allowed to run before being killed
+``geomesa.query.threads``              Integer The number of threads to use per query
+``geomesa.query.loose-bounding-box``   Boolean Use loose bounding boxes - queries will be faster but may return extraneous results
+``hbase.ranges.max-per-extended-scan`` Integer The max number of ranges used for each extended scan
```

Comment:
Should this be up with the other 'hbase' namespaces params?  Or is it lower in the list since it is obscure?

Reply:
yeah i tried to put the more important ones higher